packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -61,6 +65,37 @@ export const CallControls = (props: CallControlsProps & ContainerRectProps): JSX
     [localeStrings]
   );
 
+  /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove() */
+  const moreButtonStrings = useMemo(
+    () => ({
+      label: localeStrings.strings.call.moreButtonCallingLabel,
+      tooltipOffContent: localeStrings.strings.callWithChat.moreDrawerButtonTooltip
+    }),
+    [localeStrings]
+  );
+
+  /* @conditional-compile-remove(PSTN-calls) */ /* @conditional-compile-remove(one-to-n-calling) */
+  const holdButtonProps = usePropsFor(HoldButton);
+
+  /* @conditional-compile-remove(one-to-n-calling) */ /* @conditional-compile-remove(one-to-n-calling) */
+  const moreButtonContextualMenuItems = (): IContextualMenuItem[] => {
+    const items: IContextualMenuItem[] = [];
+
+    items.push({
```

Comment:
Just to confirm: this will be added for group calls and teams interop calls also

Reply:
I was going to bring this up in parking lot, my testing revealed to me this works in those kinds of calls to